resources/configurationFiles/analysisRNAseqGRCh38.xml

```diff
@@ -7,47 +7,50 @@
                imports="RNAseqAnalysis"
                cleanupScript="cleanupScript">
 
-        <configurationvalues>
-        <!-- Software update for the hg38 reference genome-->
-        <cvalue name='STAR_VERSION' value='2.7.6a' type='string'/>
-        <cvalue name='KALLISTO_VERSION' value='0.46.0' type='string'/>
-        <cvalue name='STAR_MODULE_NAME' value='star' type='string'/>
+    <configurationvalues>
+        <cvalue name='SPECIES' value='Homo_sapiens' type='string' />
+        <cvalue name='REF_GENOME' value='GRCh38' type='string'/>
+        <cvalue name='REF_GENOME_VERSION' value='GRCh38_decoy_ebv_phiX' type='string'/>
+        <cvalue name='GENCODE_VERSION' value='gencode_v39_chr_patch_hapl_scaff' type='string' />
 
-        <cvalue name='STAR_PARAMS_CHIMERIC' value='--chimSegmentMin 15 --chimScoreMin 1 --chimScoreJunctionNonGTAG 0 --chimJunctionOverhangMin 15  --chimSegmentReadGapMax 3 --alignSJstitchMismatchNmax 5 -1 5 5 --chimOutType Junctions SeparateSAMold' type='string'/>
-        <!-- Hg38 reference genome -->
-        <cvalue name='hg38BaseDirectory' value='/icgc/ngs_share/assemblies/hg_GRCh38' type='path'/>
-        <cvalue name='hg38DatabaseDirectory' value='${hg38BaseDirectory}/databases' type='string'/>
-        <cvalue name='hg38IndexDirectory' value='${hg38BaseDirectory}/indexes' type='string'/>
+        <cvalue name='hg38BaseDirectory' value='/omics/odcf/reference_data/by-species/${SPECIES}/${REF_GENOME}/${REF_GENOME_VERSION}' type='path' />
 
-        <cvalue name='databaseDirectory' value='${hg38DatabaseDirectory}' type='string'/>
-        <cvalue name='indexDirectory' value='${hg38IndexDirectory}' type='string'/>
+        <cvalue name='hg38BaseDirectory_legacy' value='/omics/odcf/reference_data/legacy/ngs_share/assemblies/hg_GRCh38/' type='path'/>
+        <cvalue name='hg38DatabaseDirectory_legacy' value='${hg38BaseDirectory_legacy}/databases' type='string'/>
 
-        <!--Databases and index-->
-        <cvalue name='GENOME_FA' value='${hg38BaseDirectory}/sequence/GRCh38_decoy_ebv_phiX_alt_hla_chr.fa' type='string'/>
-        <cvalue name='GENOME_GATK_INDEX' value='${hg38IndexDirectory}/bwa/bwa07_GRCh38_decoy_ebv_phiX_alt_hla_chr/GRCh38_decoy_ebv_phiX_alt_hla_chr.fa' type='string'/>
+        <cvalue name='databaseDirectory' value='${hg38DatabaseDirectory_legacy}' type='string'/>
+        <cvalue name='indexDirectory' value='${hg38BaseDirectory}' type='string'/>
 
-        <cvalue name='GENE_MODELS' value='${hg38DatabaseDirectory}/gencode/gencode31/hg38_gencode.v31.annotation.gtf' type='string'/>
-        <cvalue name='GENE_MODELS_NOGENE' value='${hg38DatabaseDirectory}/gencode/gencode31/hg38_gencode.v31.annotation.nogene.gtf' type='string' 
+        <cvalue name='GENOME_FA' value='${hg38BaseDirectory}/genome.fa' type='string'/>
+        <cvalue name='GENOME_GATK_INDEX' value='${GENOME_FA}' type='string'/>
```

Comment:
This points to the `genome.dict`, usually referred with `genome.fa`. For `GRCh38_decoy_ebv_phiX`, `genome.dict` is not yet available.